dll/cpl/desk/lang/pl-PL.rc

```diff
@@ -11,63 +11,63 @@
 
 LANGUAGE LANG_POLISH, SUBLANG_DEFAULT
 
-IDD_THEMES DIALOGEX 0, 0, 246, 210
+IDD_THEMES DIALOGEX 0, 0, 252, 218
 CAPTION "Kompozycje"
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Kompozycja to Tapeta, zestaw dźwięków, ikon i innych elementów, które pomogą Ci spersonalizować komputer za pomocą jednego kliknięcia.", IDC_STATIC, 5, 5, 235, 30
-    LTEXT "Kompozycja:", IDC_STATIC, 5, 42, 55, 10
-    COMBOBOX IDC_THEMES_COMBOBOX, 5, 52, 160, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Z&apisz jako...", IDC_THEMES_SAVE_AS, 170, 52, 70, 14
-    PUSHBUTTON "&Usuń", IDC_THEMES_DELETE, 170, 70, 70, 14
-    CONTROL "", IDC_THEMES_PREVIEW, "STATIC", SS_BITMAP, 5, 90, 235, 115, WS_EX_CLIENTEDGE
+    LTEXT "Kompozycja to tapeta, zestaw dźwięków, ikon i innych elementów, które pomogają spersonalizować komputer za pomocą jednego kliknięcia.", IDC_STATIC, 7, 5, 235, 24
+    LTEXT "Kompozycja:", IDC_STATIC, 7, 32, 55, 10
+    COMBOBOX IDC_THEMES_COMBOBOX, 7, 42, 128, 300, CBS_HASSTRINGS | CBS_AUTOHSCROLL | CBS_DROPDOWN | WS_VSCROLL | WS_TABSTOP
+    PUSHBUTTON "Z&apisz jako...", IDC_THEMES_SAVE_AS, 140, 42, 71, 14
+    PUSHBUTTON "&Usuń", IDC_THEMES_DELETE, 215, 42, 30, 14
+    CONTROL "", IDC_THEMES_PREVIEW, "STATIC", SS_BITMAP, 7, 70, 238, 141, WS_EX_CLIENTEDGE
 END
 
-IDD_BACKGROUND DIALOGEX 0, 0, 246, 210
+IDD_BACKGROUND DIALOGEX 0, 0, 252, 218
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Pulpit"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    CONTROL "", IDC_BACKGROUND_PREVIEW, "Static", SS_OWNERDRAW, 70, 10, 105, 74
+    CONTROL "", IDC_BACKGROUND_PREVIEW, "Static", SS_OWNERDRAW, 63, 7, 125, 107
     CONTROL "", IDC_BACKGROUND_LIST, "SysListView32", LVS_REPORT | LVS_SINGLESEL |
-            LVS_NOCOLUMNHEADER | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 8, 104, 174, 86
-    LTEXT "Wybierz obraz do użycia jako tapeta pulpitu", IDC_STATIC, 8, 93, 230, 9
-    PUSHBUTTON "&Przeglądaj...", IDC_BROWSE_BUTTON, 188, 104, 55, 15
-    PUSHBUTTON "&Kolor...", IDC_COLOR_BUTTON, 188, 167, 55, 15
-    LTEXT "Położenie", IDC_STATIC, 188, 128, 55, 9
-    COMBOBOX IDC_PLACEMENT_COMBO, 188, 139, 55, 54, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "&Dostosuj pulpit...", IDC_DESKTOP_CUSTOM, 8, 194, 90, 15
+            LVS_NOCOLUMNHEADER | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 7, 127, 170, 70
+    LTEXT "Wybierz obraz do użycia jako tapeta pulpitu", IDC_STATIC, 7, 115, 170, 8
+    PUSHBUTTON "&Przeglądaj...", IDC_BROWSE_BUTTON, 184, 127, 64, 14
+    PUSHBUTTON "&Kolor...", IDC_COLOR_BUTTON, 184, 187, 64, 14
+    LTEXT "Położenie", IDC_STATIC, 184, 148, 55, 8
+    COMBOBOX IDC_PLACEMENT_COMBO, 184, 158, 64, 54, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    PUSHBUTTON "&Dostosuj pulpit...", IDC_DESKTOP_CUSTOM, 7, 200, 90, 14
```

Comment:
It seems to me you're modifying the dimensions too much. These dialogs appearance should be in sync between all translations with minimal differences.

Reply:
dialogs should be bigger as in XP2k3++

Reply:
I have looked at your screenshot in PR description. If you want to make dimensions closer to Windows, you would have to do this in a separate PR, for all languages, and not mixing with translation changes. It does not make sense to do just for one translation, this would make it out of sync with other translations.

I'm going to revert these changes and leave only translations.